### PR TITLE
Issue #338 added troubleshooting info about passwords with special chars

### PR DESCRIPTION
**Pull request by thatdocslady**
Fixes issue #338 

This PR aims to provide troubleshooting info about why passwords with certain special characters fail and how to workaround.

Note: I'm basing this content on the info provided in the issue, although I'm aware that there might be some inaccuracies, let's resolve any gaps together :)

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -99,3 +101,13 @@ If you are using Windows, the problem is likely to be either an outdated version
...
+
+<a name="users-authentication"></a>
+## Users and authentication
+
+### Some special characters cause passwords to fail
+
+Depending on your operating system and shell environment, certain special characters
+can trigger variable interpolation and therefore cause passwords to fail.
+
+Workaround: When creating and entering passwords, wrap the string with single-quotes ('').
```
Comment (budhrg) [inline docs/troubleshooting.md]: `single-quotes` to `single quotes` ? and also wrap single quotes as ` ('') `

Comment (thatdocslady) [inline docs/troubleshooting.md]: aha sure, fixed now :)

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -99,3 +101,14 @@ If you are using Windows, the problem is likely to be either an outdated version
...
+<a name="users-authentication"></a>
+## Users and authentication
+
+### Some special characters cause passwords to fail
+
+Depending on your operating system and shell environment, certain special characters
+can trigger variable interpolation and therefore cause passwords to fail.
+
+Workaround: When creating and entering passwords, wrap the string with single quotes in
+the following format: '<password>'
```
Comment (budhrg) [inline docs/troubleshooting.md]: Not working due to `<`. 

It should be  `... following format: '&lt;password>'`
screenshot from 2017-02-20 16-42-18

**Comment by budhrg** [comment]
LGTM :+1:  with one suggestion

File: docs/troubleshooting.md (hunk 2 above)
Comment (thatdocslady) [inline docs/troubleshooting.md]: Hahahaha speaking of special characters ;)

**Comment by hferentschik** [comment]
It is bit oversimplified, but I went ahead and merged it. Can be refined another day.

**Pull request closed without merging**